HomeWork_03/DZ3.py

```diff
@@ -0,0 +1,79 @@
+#I love Python*42#
+
+# Zadanie 1
+x = ("I love Python")
+z = x*42
+print(z)
+
+#age_in_month#
+
+# Zadanie 2
+x = str("age_in_month")
```

Comment:
First: to create a string you don't need to call str() function. You can simply do like this `x = "age_in_month").
Secont: to create a variable named `age_in_month` you should write a name of a variable and than assign some value to it. For example, `age_in_month = 12`. Rewrite this piece of code according to the task